src/grid/index.js

```diff
@@ -164,15 +179,25 @@ class Grid {
       return false;
     }).map(size => getCleanSize(size));
 
-    for (i = 0, len = sizeList.length; i < len; i++) {
+    len = sizeList.length;
+    if(tracks === "none"){
+      len = this.getProps("maxTracks");
+      console.log(len);
+      sizeList = 'auto,'.repeat(len).split(",");
```

Comment:
@siddharthapaul001 Mason is intended to be used even in IE8. `String#repeat` is not supported in any version on IE at all.

Reply:
Used another alternative.